### DISCUSSION TITLE
The file cperl-indent-styles.pl is missing

### DISCUSSION THREAD

**Discussion by benkasminbullock** [HaraldJoerg/cperl-mode#14] [Q&A] [answered]
The file cperl-indent-styles.pl is used by this test:

https://github.com/HaraldJoerg/cperl-mode/blob/master/test/lisp/progmodes/cperl-mode-tests.el#L180

But it doesn't seem to exist:

https://github.com/HaraldJoerg/cperl-mode/tree/master/test/lisp/progmodes/cperl-mode-resources

**Comment by benkasminbullock** [comment, marked as answer]
Never mind, I found it:

https://github.com/emacs-mirror/emacs/blob/master/test/lisp/progmodes/cperl-mode-resources/cperl-indent-styles.pl

https://github.com/emacs-mirror/emacs/blob/master/test/lisp/progmodes/cperl-mode-resources/cperl-indent-exp.pl

Thanks.

**Reply by HaraldJoerg** [reply]
Well, of course it should be in the repository if the tests are using it.  I've pushed it now to the master and upstream branches.  Thanks for the notice!
